.github/workflows/ci.yml

```diff
@@ -229,7 +229,11 @@ jobs:
           # aren't too long. On Windows tox won't pass it through so it has no
           # effect. On Linux it doesn't make a difference one way or another.
           TMPDIR: "/tmp"
-        run: tox -e integration
+        run: |
+          # Run with Foolscap forced:
+          __TAHOE_INTEGRATION_FORCE_FOOLSCAP=1 tox -e integration
+          # Run with Foolscap not forced, which should result in HTTP being used.
+          __TAHOE_INTEGRATION_FORCE_FOOLSCAP=0 tox -e integration
```

Comment:
I suppose it would probably be nice to split these runs across two different jobs so that it's a bit easier to tell at a glance if there is a Foolscap-only or HTTP-only problem with them.

Also I am not much of a fan of tox but since we're using it, I wonder if setting these environment variables in a tox environment instead of in the GitHub Actions config would be preferable.  I guess this might look something like `tox -e integration-foolscap` replacing the first tox command and maybe just `tox -e integration` for the second?  (Or maybe it is better to be explicit in the latter case as well?  But I'm not sure what that leaves `tox -e integration` meaning.)

This presumably makes it slightly easier for devs to do local test runs for these cases.

Reply:
I do think it's best to encode this in Tox rather than _just_ GHA ... but also what about it just being an option to the tests instead of an environment variable?

That is, so it's `py.test .... --foolscap-only` (or I suppose `py.test ... --http-only` if that's the intent)

Reply:
OK, tried a different approach.